test/commands/diff.test.ts

```diff
@@ -105,6 +105,35 @@ DevCycle Variable Changes:
 	   Location: [test/utils/diff/sampleDiff.js:L1](https://example.com/files#diff-c197a837fe3ee51fcef381dc90df1cde5c759ad43f47c0cb72968af943205fa3L1)
 `
 
+const linkedBitbucketExpected = `
+DevCycle Variable Changes:
+
+🟢  6 Variables Added
+🔴  1 Variable Removed
+
+🟢 Added
+
+  1. simple-case
+	   Location: [test/utils/diff/sampleDiff.js:L1](https://bitbucket.org/devcyclehq/test-code-refs-pipes/pull-requests/7#Ltest/utils/diff/sampleDiff.jsT1)
+  2. duplicate-case
+	   Locations:
+	    - [test/utils/diff/sampleDiff.js:L2](https://bitbucket.org/devcyclehq/test-code-refs-pipes/pull-requests/7#Ltest/utils/diff/sampleDiff.jsT2)
+	    - [test/utils/diff/sampleDiff.js:L3](https://bitbucket.org/devcyclehq/test-code-refs-pipes/pull-requests/7#Ltest/utils/diff/sampleDiff.jsT3)
+  3. single-quotes
+	   Location: [test/utils/diff/sampleDiff.js:L5](https://bitbucket.org/devcyclehq/test-code-refs-pipes/pull-requests/7#Ltest/utils/diff/sampleDiff.jsT5)
```

Comment:
these URLs look kinda wack are you sure this is their format?

Reply:
they are indeed whacky, but they are correct
https://bitbucket.org/devcyclehq/test-code-refs-pipes/pull-requests/7#Lexample/js/src/main.jsT33

looks like it works with or without the `L` after the `#` though